wgsl/index.bs

```diff
@@ -5583,7 +5583,7 @@ The expression must evaluate to a reference with an [=integer scalar=] [=store t
   <tr algorithm="decrement statement">
     <td class="nowrap">|r| : ref&lt;<var ignore>SC</var>,|T|,[=access/read_write=]&gt;,<br>
         |T| is [=integer scalar=]<br>
-    <td class="nowrap">|r|--
+    <td class="nowrap">|r|&#45;&#45;
```

Comment:
I suggest using backticks instead.

See https://github.com/gpuweb/gpuweb/pull/2576#issuecomment-1034377312

Reply:
@dneto0 thanks for the feedback, I agree that this looks the best out of three, and I applied the change.